pkg/network/vmispec/hotplug.go

```diff
@@ -69,3 +69,15 @@ func NetworksToHotplugWhosePodIfacesAreReady(vmi *v1.VirtualMachineInstance) []v
 
 	return networksToHotplug
 }
+
```

Comment:
Please fix the commit message.

Reply:
Whats wrong with it?

Reply:
> Filter interfaces with bridge binding only for hot{unp}plug operations as its the
only supposed binding at the moment.

This ^^

Reply:
Done